types/defines/ai.d.ts

```diff
@@ -724,6 +725,6 @@ export declare abstract class Ai<AiModelList extends AiModelListType = AiModels>
     model: Name,
     inputs: AiModelList[Name]["inputs"],
     options?: AiOptions,
-  ): Promise<AiModelList[Name]["postProcessedOutputs"]>;
+  ): Promise<AiModelList[Name]["postProcessedOutputs"]| Response>;
```

Comment:
maybe make the output depend on the `returnRawResponse` option

```ts
run<Name extends keyof AiModelList>(
  model: Name, 
  inputs: AiModelList[Name]["inputs"], 
  options?: AiOptions & { returnRawResponse?: boolean }
): Promise<
  options extends { returnRawResponse: true } 
    ? Response 
    : AiModelList[Name]["postProcessedOutputs"]
>;
```

Reply:
I have updated this, thanks for the review!